release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md

```diff
@@ -0,0 +1,12 @@
+## 2021-02-03 Version 1.13.0.0
+Initial launch of Asynchronous Search on opendistro-for-elasticsearch
```

Comment:
can we put a brief explanantion of what it does and we can also link a reference to detailed RFC/Document

Reply:
added the same.